tools/bl.c

```diff
@@ -693,6 +725,10 @@ int main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+	/* All valid comments using device path have more than 2 arguments */
+	if (argc > ARG__COUNT)
+		bl_auto_dev(argv);
```

Comment:
Cool.  Probably worth doing this in the calibrate tool too.  However, the argument we want to change there is at a different offset.

Reply:
hmm, I didn't notice dev_node is also used in calibrate... will refactor a little bit, and apply to both tools